components/organisms/Layout.js

```diff
@@ -32,9 +38,50 @@ export const Layout = ({
             headerLogoImage="/sig-blk-en.svg"
           />
         </div>
-
         <div className="mb-2 border-t pb-2 mt-4"></div>
-        <nav className="layout-container">Menu</nav>
+
+        {menu ? (
+          <nav
+            title="Menu"
+            className="lg:container lg:mx-auto lg:px-6 lg:justify-end lg:flex"
+            data-cy="menu"
+          >
+            <div className="layout-container pb-4">
+              <button
+                onClick={onMenuClick}
+                id="menuButton"
+                className="text-h4 focus:outline-none focus:ring-2 focus:ring-black"
+              >
```

Comment:
I think this should be properly hidden when not in use. Right now, it still takes up space and padding.